<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1411</width>
    <height>909</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload .onnx Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Output Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>381</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="uploadStatusLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose a .onnx Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose a output path</string>
    </property>
   </widget>
   <widget class="QLabel" name="convertStatusLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>1331</width>
      <height>251</height>
     </rect>
    </property>
    <property name="text">
     <string>Model Convert Status</string>
    </property>
   </widget>
   <widget class="QComboBox" name="encodeChoiceComboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="encodeStatusLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Disable Encoding</string>
    </property>
   </widget>
   <widget class="QComboBox" name="quantizeChoiceComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Yes</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="quantizeStatusLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantize Your Model: Yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quantizeButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>380</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Quantize Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="quantizeImageStatusLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantize Model Input (.jpg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Model Conversion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>STEP1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>STEP2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>STEP3:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>STEP4:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>STEP5:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>STEP1:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadDLCButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload .dlc Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Model Visualization</string>
    </property>
   </widget>
   <widget class="QLabel" name="uploadDLCStatusLabel">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose a .dlc Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>STEP2:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualizeButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize</string>
    </property>
   </widget>
   <widget class="QLabel" name="visualizeStatusLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>561</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>Model Visualize Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadEncodeButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload .encodings File</string>
    </property>
   </widget>
   <widget class="QLabel" name="uploadEncodeStatusLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose a .encodings File</string>
    </property>
   </widget>
   <widget class="QComboBox" name="resolutionChoiceComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>480x480</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>640x640</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="resolutionStatusLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1411</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
